### PR TITLE
I sorta halfway made the menu screen

### PR DESCRIPTION
**Pull request by chubbibunniomnomnom**

### CODE REVIEW

**Comment by sgallese** [comment]
Descriptive git commit titles are helpful :)

File: SheepDash/core/src/com/missionbit/game/sprites/Animation.java [hunk 1]
```diff
@@ -0,0 +1,40 @@
+package com.missionbit.game.sprites;
+
+import com.badlogic.gdx.graphics.g2d.TextureRegion;
+import com.badlogic.gdx.utils.Array;
+
+/**
+ * Created by MissionBit on 6/26/17.
+ */
+
+public class Animation {
```
Comment (sgallese) [inline SheepDash/core/src/com/missionbit/game/sprites/Animation.java:10]: required?

File: SheepDash/core/src/com/missionbit/game/sprites/Sheep.java [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+import com.badlogic.gdx.graphics.Texture;
+import com.badlogic.gdx.math.Rectangle;
+import com.badlogic.gdx.math.Vector3;
+
+/**
+ * Created by MissionBit on 6/27/17.
+ */
+
+public class Sheep {
```
Comment (sgallese) [inline SheepDash/core/src/com/missionbit/game/sprites/Sheep.java:11]: This used elsewhere?  If yes, would you want an abstract class or interface for the animal?

File: SheepDash/core/src/com/missionbit/game/states/MenuState.java [hunk 3]
```diff
@@ -1,8 +1,46 @@
...
+    private Texture playBtn;
+
+    public MenuState(GameStateManager gsm) {
+        super(gsm);
+        background = new Texture("MenuScreen.png");
+        playBtn = new Texture("PlayBtn.png");
+    }
+
+    @Override
+    protected void handleInput() {
```
Comment (juemura) [inline SheepDash/core/src/com/missionbit/game/states/MenuState.java:24]: this needs to be public

File: SheepDash/core/src/com/missionbit/game/states/State.java [hunk 4]
```diff
@@ -8,10 +8,6 @@
  * Created by missionbit on 6/26/17.
  */
 
-public abstract class State {
-    protected OrthographicCamera cam;
-    protected Vector3 mouse;
-    protected GameStateManager gsm;
```
Comment (juemura) [inline SheepDash/core/src/com/missionbit/game/states/State.java:11]: Not sure why these lines were removed, but we need them. You won't be able to compile without these.

File: SheepDash/core/src/com/missionbit/game/states/State.java [hunk 5]
```diff
@@ -8,10 +8,6 @@
...
 
     protected State(GameStateManager gsm) {
```
Comment (juemura) [inline SheepDash/core/src/com/missionbit/game/states/State.java:12]: this should be public

**Review by juemura — CHANGES_REQUESTED** [review]
I added some comments on things that need to be changed for the code to compile. Everything else looks good! 
Once you commit these changes I can merge. No need to create another pull request (once you push to the same branch, this PR will be updated automatically). 
Good job!

**Pull request merged**